projects/fsastorefrontlib/src/cms-components/my-account/inbox/unread-messages-indicator/unread-messages-indicator.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="message-indicator-container">
+  <a
+    [routerLink]="
+      {
+        cxRoute: 'inbox'
+      } | cxUrl
+    "
+    class="messages-indicator"
+    *ngIf="user$ | async"
+  >
+    <div class="fs-icon icon-envelope"></div>
+    <ng-container *ngIf="numberOfUnreadMessages$ | async as numberOfMessages">
```

Comment:
numberOfMessages would imply all messages, I would suggest to stick to original naming from observable -> numberOfUnreadMessages